src/sentry/models/notificationsetting.py

```diff
@@ -47,8 +74,21 @@ def value_str(self) -> str:
     def provider_str(self) -> str:
         return get_provider_name(self.provider)
 
+
+@region_silo_only_model
+class NotificationSetting(Model, NotificationSettingBase):
+    """
+    A setting of when to notify a user or team about activity within the app.
+    Each row is a notification setting where a key is:
+    ("scope_type", "scope_identifier", "target", "provider", "type"),
+    and the value is ("value").
+    """
+
+    __include_in_export__ = False
```

Comment:
Hmm, should this possibly be included in export?  I don't know if there is a good test for this feature, but essentially I believe exported models are part of data migration or account migration tooling?  Maybe someone else knows this deeply?